Modules/_json.c

```diff
@@ -1267,10 +1286,21 @@ encoder_call(PyEncoderObject *self, PyObject *args, PyObject *kwds)
     _PyUnicodeWriter_Init(&writer);
     writer.overallocate = 1;
 
-    if (encoder_listencode_obj(self, &writer, obj, indent_level)) {
+    PyObject * current_newline_indent = NULL;
+    if (self->indent != Py_None) {
+        current_newline_indent = _create_newline_indent(self->indent,
+                                                        indent_level);
+        if (current_newline_indent == NULL) {
+            _PyUnicodeWriter_Dealloc(&writer);
+            return NULL;
+        }
+    }
+    if (encoder_listencode_obj(self, &writer, obj, indent_level,
```

Comment:
Do we need to pass `indent_level` if we pass `current_newline_indent`? I suppose that it is no longer used.